packages/tradershub/src/hooks/usePlatformAssets.ts

```diff
@@ -0,0 +1,60 @@
+import { useEffect } from 'react';
+import { useExchangeRates, useTradingAccountsList } from '@deriv/api';
+import { BrokerCodes, Regulation } from '../constants/constants';
+
+/**
+ * @description This hook is used to get the total demo and real balance of the user
+ * @param regulation  - Regulation of the user
+ */
+const usePlatformAssets = (regulation?: string) => {
+    const {
+        data: tradingAccount,
+        fiat_account: fiatAccount,
```

Comment:
This name is misleading, we should change it to `fiat_account_currency` or `fiat_currency` but it is out of the scope for this PR, just change the name you are using in this file
```suggestion
        fiat_account: fiatCurrency,
```